lesson_07/dags/sql/transfer_from_dwh_bronze_to_dwh_silver_customers.sql

```diff
@@ -0,0 +1,30 @@
+TRUNCATE TABLE `{{ params.project_id }}.silver.customers`
```

Comment:
At the moment you're truncating the table users don't have access to the data. This may be a trouble. 
I recommend truncating partitions only, not the whole tables. Pls Read about MERGE statement in SQL and what it can do